target/linux/ramips/dts/mt7621_netgear_r6220.dts

```diff
@@ -1,14 +1,40 @@
 // SPDX-License-Identifier: GPL-2.0
 /dts-v1/;
 
-#include "mt7621_netgear_r6220.dtsi"
-
-#include <dt-bindings/gpio/gpio.h>
-#include <dt-bindings/input/input.h>
+#include "mt7621_netgear_r.dtsi"
 
 / {
 	compatible = "netgear,r6220", "mediatek,mt7621-soc";
 	model = "Netgear R6220";
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 7 GPIO_ACTIVE_HIGH>;
```

Comment:
Despite having an additional button, all Netgear R devices have the same GPIOs for wps and reset, but R6220/WNDR3700v5 are GPIO_ACTIVE_HIGH, while the others are GPIO_ACTIVE_LOW. Is this correct or a mistake?